interfaces/cython/cantera/examples/onedim/premixed_counterflow_twin_flame.py

```diff
@@ -87,6 +87,9 @@ def solveOpposedFlame(oppFlame, massFlux=0.12, loglevel=1,
 # Create the flame object
 oppFlame = ct.CounterflowTwinPremixedFlame(gas, width=width)
 
+# Use mixture-averaged properties
+oppFlame.transport_model = 'Mix'
```

Comment:
Correct. In Reinserted in [e738001](https://github.com/Cantera/cantera/pull/356/commits/e73800105c545144f2f6e73102fc255a1d9383a5) I have commented this out and instead provided an option to use the multicomponent formulation if a user so desires.